src/assets/translations/en/main.json

```diff
@@ -857,11 +857,12 @@
   },
   "walletProvider": {
     "errors": {
-      "walletNotFound": "Unable to communicate with your wallet"
+      "walletNotFound": "Unable to communicate with your wallet. If problem persists, please contact ShapeShift support."
```

Comment:
shapeshift support does not support keepkey - this needs to be keepkey support

Reply:
Changed